Unified/closor.py

```diff
@@ -499,7 +499,7 @@ def close(self):
                         ## make the specific relval rules and the replicas
                         ## figure the destination(s) out
                         destinations = set()
-                        if tier != "RECO" and tier != "ALCARECO":
+                        if tier != "RECO" and tier != "ALCARECO" and tier != "NANOAOD" and tier != "NANOAODSIM":
```

Comment:
Might suggest having a configuration variable with a list and `tier not in x`
I suspect we will expand the list incrementally

Reply:
Hi @nsmith- Thanks for commenting. In the PR comment I was suggesting to use the same variable from unified config as for the standard workflows but it doesn't seem right to mix the two things. I will create a separate one for RalVal.

Reply:
@nsmith- I just added two separate configuration variables for RelVal and nonRelVal workflows.